setup.cfg

```diff
@@ -1,3 +1,43 @@
 [metadata]
 version = 0.6.2
+name = svgcheck
+description = Verify that an svg file is compliant with the RFC standards.
+long_description = file:  README.md, LICENSE
+long_description_content_type = text/markdown
+keywords = svg, validation, rfc
+author = Jim Schaad
+author_email = ietf@augustcellars.com
```

Comment:
We should replace author and author_email with maintainer information here.

Reply:
There are also `maintainer` and `maintainer_email` settings.

Reply:
I have changed `author` to `IETF Tools Team` and removed the `author_email`.